docs/src/SUMMARY.md

```diff
@@ -19,7 +19,7 @@
   - [Install Cluster API for Oracle Cloud Infrastructure](./gs/install-cluster-api.md)
   - [Create Workload Cluster](./gs/create-workload-cluster.md)
   - [Create Workload Templates](./gs/create-workload-templates.md)
-  - [Using externally managed infrastructure](./gs/externally-managed-infrastructure.md)
+  - [Using externally managed infrastructure](./gs/externally-managed-cluster-infrastructure.md)
```

Comment:
These files shouldn't be changed. I'll have to look into why they are being updated.